swift/Arrow/Sources/Arrow/ArrowArray.swift

```diff
@@ -17,7 +17,60 @@
 
 import Foundation
 
-public class ArrowArray<T> {
+public class ArrowArrayHolder {
+    public let type: ArrowType.Info
+    public let length: UInt
+    public let nullCount: UInt
+    public let holder: Any
```

Comment:
How about renaming this to `array`?

```suggestion
    public let array: Any
```

I feel that `ArrowArrayHolder.holder` is strange.

Reply:
Gotcha, yeah, good catch, I will update.